tests/ui/needless_partial_ord_impl.stderr

```diff
@@ -0,0 +1,28 @@
+error: manual implementation of `PartialOrd` when `Ord` is already implemented
+  --> $DIR/needless_partial_ord_impl.rs:18:1
+   |
+LL | /  impl PartialOrd for A {
+LL | |      fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+   | | _____________________________________________________________-
+LL | ||         todo!();
+LL | ||     }
+   | ||_____- help: change this to: `{ Some(self.cmp(self)) }`
```

Comment:
Shouldn't this be `{ Some(self.cmp(other)) }`?

[`std::cmp::Ord`](https://doc.rust-lang.org/std/cmp/trait.Ord.html) is derived to use `std::cmp::Ord::cmp(&self, &other)`. So  I think it should actually use the second argument (there's a suggestion about this in the main lint file)